services/QuillLMS/app/models/concerns/teacher.rb

```diff
@@ -323,22 +323,23 @@ def classroom_units(includes_value = nil)
   def update_teacher params
     return if !teacher?
     params[:role] = 'teacher' if params[:role] != 'student'
-    params.permit(:id,
-                  :name,
-                  :role,
-                  :username,
-                  :authenticity_token,
-                  :email,
-                  :time_zone,
-                  :password,
-                  :school_options_do_not_apply,
-                  :school_id,
-                  :send_newsletter,
-                  :original_selected_school_id,
-                  :google_id,
-                  :clever_id,
-                  :signed_up_with_google,
-                  :post_google_classroom_assignments)
+    params.permit(
+      :id,
+      :name,
+      :role,
+      :username,
+      :authenticity_token,
+      :email,
+      :time_zone,
+      :password,
+      :school_options_do_not_apply,
+      :school_id,
+      :send_newsletter,
+      :original_selected_school_id,
+      :google_id,
+      :clever_id,
+      :signed_up_with_google
+    )
```

Comment:
I think we should leave this in for now (and remove this on the second deploy), otherwise forms that are loaded on the page pre-deploy will error (Unpermitted param) if submitted post-deploy.

Reply:
Great point.  I will move this to part 2